doc/circuitikzmanual.tex

```diff
@@ -441,14 +441,14 @@ \subsection{Getting started with \Circuitikz: a current shunt}
 
 Let's start with the first type of component, and build a basic mesh:
 
-\begin{LTXexample}[varwidth=true]
+\begin{ LTXexample}[varwidth=true]
```

Comment:
...and yes, you added the problematic space here. ;-)

Reply:
You could have noticed that by these markers: the red one means that the manual is not compiling correctly.

![image](https://user-images.githubusercontent.com/6414907/170883389-42d5ea8f-0325-4407-8baf-16a0cbd42091.png)